lib/src/component_declaration/flux_component.dart

```diff
@@ -140,9 +132,9 @@ abstract class _FluxComponentMixin<TProps extends FluxUiProps> implements Batche
     return {};
   }
 
-  /// Register a [subscription] that should be canceled when the component unmounts.
-  ///
-  /// Cancellation will be handled automatically by [componentWillUnmount].
+  /// Register a [subscription] that should be canceled when the component
+  /// unmounts. Cancellation will be handled automatically by
+  /// [componentWillUnmount].
```

Comment:
Seems like the version of this file was more up to date in over_react (i.e., these changes should be discarded)